Agreements/Airport/GD_FF.txt

```diff
@@ -0,0 +1,2 @@
+COPX:CDF:*:*:EGFF:12:GDAPP:FFAPP:*:8000:|HDG
+COPX:CDF:*:*:EGFF:30:GDAPP:FFAPP:*:3500:|HDG
```

Comment:
suggest moving CDF to the fourth column - as it is the next fix - rather than fixbefore